util/OptionsDB.cpp

```diff
@@ -431,6 +431,24 @@ void OptionsDB::SetFromCommandLine(const std::vector<std::string>& args) {
     }
 }
 
+void OptionsDB::SetFromFile(const boost::filesystem::path& file_path,
+                            const std::string& version) {
```

Comment:
The FO Code Standards wiki says to follow your preference, as long as it doesn't contradict the style of the rest of the file.

Might be useful to mention a preferred (or prevalent) style in the Wiki, when it isn't obvious from a given file.